docs/pages/7_0_0_release_notes.md

```diff
@@ -143,30 +190,51 @@ node.descendants(ASTMethodCall.class)
     .foreach(System.out::println);
 ```
 
-A pipeline like shown here traverses the tree lazily, which is more efficient than traversing eagerly to put all descendants in a list. It is also much easier to change than the old imperative way.
+A pipeline like shown here traverses the tree lazily, which is more efficient than traversing eagerly to put all
+descendants in a list. It is also much easier to change than the old imperative way.
 
-To make this API as accessible as possible, the {% jdoc core::lang.ast.Node %} interface has been fitted with new methods producing node streams. Those methods replace previous tree traversal methods like `Node#findDescendantsOfType`. In all cases, they should be more efficient and more convenient.
+To make this API as accessible as possible, the {% jdoc core::lang.ast.Node %} interface has been fitted with new
+methods producing node streams. Those methods replace previous tree traversal methods like `Node#findDescendantsOfType`.
+In all cases, they should be more efficient and more convenient.
 
 See {% jdoc core::lang.ast.NodeStream %} for more details.
 
+Contributors: [Clément Fournier](https://github.com/oowekyala) (@oowekyala)
 
 #### JavaScript support
 
 The JS specific parser options have been removed. The parser now always retains comments and uses version ES6.
 The language module registers only one version (as before), now correctly with version "ES6" instead of "3".
-Since there is only one version available for JavaScript there is actually no need to selected a specific version.
+Since there is only one version available for JavaScript there is actually no need to select a specific version.
 The default version is always ES6.
```

Comment:
most of this is no longer true after #4387, in particular https://github.com/pmd/pmd/pull/4387/files#diff-6a73814684c68604f37a965213cbfa5e3c0bbf7f4d5d075de9f2f987682122d3

Reply:
That's already addressed by the very same PR (https://github.com/pmd/pmd/pull/4387/files#diff-2fdc02f732cb8c6cc158d3e9cf5c05566e942377d5bd1761b45f9479bf3f811e) - I'll rebase this PR soon, then this section should read differently.